cranelift/filetests/filetests/runtests/x64-xmm-mem-align-bug.clif

```diff
@@ -0,0 +1,17 @@
+test run
+set enable_llvm_abi_extensions
+target x86_64
```

Comment:
Should we just run this on all architectures as well as the interpreter? The bug was x64-specific but the test should work everywhere and it would be nice to know if it didn't.

Reply:
As a further suggestion, we can also remove the `set enable_llvm_abi_extensions` since we no longer have `i128`'s in this new minimized test case.